portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase

```diff
@@ -19,58 +19,82 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could delete a Custom Export."
 	@priority = "4"
 	test DeleteCustomExportProcess {
-		LAR.exportSiteCP(
-			larFileName = "CustomExport",
-			siteScopeName = "Guest");
+		task ("Create a Custom Export") {
+			LAR.exportSiteCP(
+				larFileName = "CustomExport",
+				siteScopeName = "Guest");
+		}
 
-		LAR.deleteCurrentAndPreviousProcess(larFileName = "CustomExport");
+		task ("Delete the created Custom Export") {
+			LAR.deleteCurrentAndPreviousProcess(larFileName = "CustomExport");
+		}
 
-		AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1");
+		task ("Assert that the Custom Export was deleted") {
+			AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS_1");
+		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Site admin should be able to delete a Export on the Documents and Media Portlet."
 	@priority = "4"
 	test DeletePortletExportProcess {
-		Navigator.openURL();
+		task ("Go to Content & Data --> Documents and Media and Export a Document and Media") {
+			Navigator.openURL();
 
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Documents and Media");
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Documents and Media");
 
-		LAR.exportPortlet(larFileName = "DMExport");
+			LAR.exportPortlet(larFileName = "DMExport");
+		}
 
-		Refresh();
+		task ("Delete the Export") {
+			Refresh();
 
-		LAR.deleteCurrentAndPreviousProcess(
-			deleteFromPortlet = "true",
-			larFileName = "DMExport");
+			LAR.deleteCurrentAndPreviousProcess(
+				deleteFromPortlet = "true",
+				larFileName = "DMExport");
+		}
 
-		AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS");
+		task ("Assert that the Export was deleted") {
+			AssertElementNotPresent(locator1 = "ExportImport#CURRENT_AND_PREVIOUS_STATUS");
+		}
 	}
 
+	@description = "This is a use case for LRQA-41349. Verify that a site admin could relaunch a Custom Export."
```

Comment:
No need to use uppercase letters in custom export. 
As this is a process you retry to do again, you should use the word "process" at the end of the sentence.